pinot-integration-tests/src/test/java/com/linkedin/pinot/integration/tests/DeleteAPIHybridClusterIntegrationTest.java

```diff
@@ -0,0 +1,324 @@
+        forSegmentListAPIWithTableType(TABLE_NAME, CommonConstants.Helix.TableType.REALTIME.toString()));
+    JSONArray realtimeSegmentsList = getSegmentsFromJsonSegmentAPI(segmentList, CommonConstants.Helix.TableType.REALTIME.toString());
+
+    String removedSegment = realtimeSegmentsList.get(0).toString();
+    long removedSegmentRows = getNumRowsFromRealtimeMetadata(removedSegment);
+    Assert.assertNotSame(removedSegmentRows, 0L);
+
+    sendGetRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).
+        forDeleteSegmentWithGetAPI(TABLE_NAME, removedSegment, CommonConstants.Helix.TableType.REALTIME.toString()));
+
+    waitForNumRows(currRealtimeRows - removedSegmentRows, CommonConstants.Helix.TableType.REALTIME);
+
+    String postDeleteSegmentList = sendGetRequest(
+        ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forSegmentListAPIWithTableType(TABLE_NAME, CommonConstants.Helix.TableType.REALTIME.toString()));
+    JSONArray realtimeSegmentsListReturn = getSegmentsFromJsonSegmentAPI(postDeleteSegmentList, CommonConstants.Helix.TableType.REALTIME.toString());
+    realtimeSegmentsList.remove(removedSegment);
+    Assert.assertEquals(realtimeSegmentsListReturn, realtimeSegmentsList);
+  }
+
+  @Test
+  public void deleteRealtimeSegmentFromDeleteAPI() throws Exception {
+    long currRealtimeRows = getNumRows(CommonConstants.Helix.TableType.REALTIME);
+
+    String segmentList = sendGetRequest(
+        ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forSegmentListAPIWithTableType(TABLE_NAME, CommonConstants.Helix.TableType.REALTIME.toString()));
+    JSONArray realtimeSegmentsList =  getSegmentsFromJsonSegmentAPI(segmentList, CommonConstants.Helix.TableType.REALTIME.toString());
+
+    String removedSegment = realtimeSegmentsList.get(0).toString();
+    long removedSegmentRows = getNumRowsFromRealtimeMetadata(removedSegment);
+    Assert.assertNotSame(removedSegmentRows, 0L);
+
+    sendDeleteRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).
+        forSegmentDeleteAPI(TABLE_NAME, removedSegment, CommonConstants.Helix.TableType.REALTIME.toString()));
+
+    waitForNumRows(currRealtimeRows - removedSegmentRows, CommonConstants.Helix.TableType.REALTIME);
+
+    String postDeleteSegmentList = sendGetRequest(
+        ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).forSegmentListAPIWithTableType(TABLE_NAME,
+            CommonConstants.Helix.TableType.REALTIME.toString()));
+    JSONArray realtimeSegmentsListReturn = getSegmentsFromJsonSegmentAPI(postDeleteSegmentList, CommonConstants.Helix.TableType.REALTIME.toString());
+    realtimeSegmentsList.remove(removedSegment);
+    Assert.assertEquals(realtimeSegmentsListReturn, realtimeSegmentsList);
+  }
+
+  // @Test TODO: Add back when we use LLC only
+  public void deleteAllRealtimeSegmentsFromGetAPI() throws Exception {
+    sendGetRequest(ControllerRequestURLBuilder.baseUrl(CONTROLLER_BASE_API_URL).
```

Comment:
I suggest you remove the body of these methods. we can re-implement them when we set the test to run with llc